packages/webdriver/src/request.ts

```diff
@@ -67,24 +86,26 @@ export default class WebDriverRequest extends EventEmitter {
         if (this.body && (Object.keys(this.body).length || this.method === 'POST')) {
             const contentLength = Buffer.byteLength(JSON.stringify(this.body), 'utf8')
             requestOptions.json = this.body
-            requestOptions.headers['Content-Length'] = contentLength
+            requestOptions.headers!['Content-Length'] = `${contentLength}`
```

Comment:
how can we be sure `headers` always exists in the `requestOptions`?

Reply:
it's defined in line 75

Reply:
Do you think it's ok to use `!` here? If not, is there a way to overcome it? If no let's keep it as is

Reply:
I think it is ok as there is no chance `headers` is undefined.